migrate_async_to_gr.yml

```diff
@@ -0,0 +1,78 @@
+#
+# This playbook migrates the primary-replicas setup generated for mysql1,mysql2,mysql3 servers into a Group Replication Cluster
+#
+
+## Settings for ProxySQL tutorial
+- hosts: mysql1
+  gather_facts: false
+  tasks:
+
+  - name: Create GR user
+    shell: >
+      mysql -uroot -e "CREATE USER IF NOT EXISTS 'grdr'@'10.%' IDENTIFIED BY 'R3pl1cat10n#';
+      GRANT REPLICATION SLAVE, CONNECTION_ADMIN, BACKUP_ADMIN, GROUP_REPLICATION_STREAM ON *.* TO 'grdr'@'10.%';"
+
+- hosts: mysql2:mysql3
+  gather_facts: false
+  tasks:
+
+  - name: Reset master, purge binlogs
+    shell: >
+      mysql -uroot -e "STOP SLAVE; RESET SLAVE ALL;"
```

Comment:
STOP REPLICA; RESET REPLICA ALL;

Reply:
Should this also do RESET SOURCE/MASTER; ?

Reply:
I checked but it is not needed. Tested end-to-end and works fine as is.